parsec/backend/postgresql/realm_queries/maintenance.py

```diff
@@ -38,7 +38,6 @@
     RealmStatus,
 )
 
-
```

Comment:
Why do you change the linter in the same PR ?
Many files are modified for no reason :/

Reply:
Should be better now